episodes/formative-assessment.md

```diff
@@ -201,18 +201,18 @@ n = -100
 _What is the result at the end of the above assignments?_
 
 a) n = 300 and m = 300
-b) n = -100 and m = 300
-c) n = -100 and m = -100
-d) n = 200 and m = -100
+b) n = -100 and m = -100
+c) n = 200 and m = 200
+d) n = -100 and m = 300
 
-The correct answer is b., while plausible distractors identify different misconceptions:
+The correct answer is d., while plausible distractors identify different misconceptions:
 
-a\) they understand the value held by `n` is now also held by `m` but do not understand the value of `n` has 
+a) they understand the value held by `n` is now also held by `m` but do not understand the value of `n` has 
 been updated since the initial assignment (e.g. treating it as a constant).
-c\) they do not understand `n` and `m` are separate variables/containers for values (which can hold the same values) 
+b) they do not understand `n` and `m` are separate variables/containers for values (which can hold the same values) 
 but rather assume that `m` is referencing variable `n` instead of copying its value at the time of assignment 
 (e.g. behaving as a reference to another cell in a spreadsheet).
-d\) they think that the reassignment for `n` actually subtracts `100` from the original value.
+c) they think that the reassignment for `n` actually subtracts `100` from the original value.
```

Comment:
```suggestion
c) they think that the reassignment for `n` actually subtracts `100` from the original value and that it also propagates to the value of `m` (which they treat as a reference to `n` rather than a separate value).
```

Reply:
Felt like we needed more explanation in c) as options b) and c) are now very similar.

Reply:
Do you think it would be better to change that incorrect answer to

> n = 200 and m = 300

because this would not combine two misconceptions?

Reply:
I have changed m to 300, to avoid conflating the two misconceptions.